packages/dashboard/lib/DashboardServer.ts

```diff
@@ -94,6 +95,30 @@ export class DashboardServer {
       this.expressApp.post("/rpc", this.postRpc.bind(this));
     }
 
+    this.expressApp.get("/analytics", (_req, res) => {
+      const userConfig = Config.getUserConfig();
+      res.json({
+        enableAnalytics: userConfig.get("enableAnalytics"),
+        analyticsSet: userConfig.get("analyticsSet"),
+        analyticsMessageDateTime: userConfig.get("analyticsMessageDateTime")
+      });
+    });
+
+    this.expressApp.put("/analytics", (req, _res) => {
+      const { enable } = req.body as { enable: boolean };
```

Comment:
Could we change this to `value` or something? Or maybe separate things like `{ enable: true } | { disable: true } | {}`?

Maybe this is a nit considering the urgency. Not requesting changes for this one... I just think `{ enable: false }` is like... "so you want me to... do nothing?"

Reply:
👍 I've renamed it to `value`